### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by renatamarques97**
resolves issue #23

I created controllers, views, routes, js, css and specs for home page.

image

### CODE REVIEW

File: spec/routing/pages_routing_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+RSpec.describe PagesController, type: :routing do
+  describe "routing" do
+    it "routes to #home" do
+      expect(:get => "/").to route_to("pages#home")
```
Comment (marclerodrigues) [inline spec/routing/pages_routing_spec.rb]: Ruby 1.9 hash syntax.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Users", type: :request do
   describe "GET /users" do
-    it "works! (now write some real specs)" do
+    it "works!" do
```
Comment (marclerodrigues) [inline spec/requests/users_spec.rb]: `it "returns the correct response code" do`.

Always add meaningful messages.

File: spec/requests/pages_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'home page', type: :request do
+  describe "GET /root" do
```
Comment (marclerodrigues) [inline spec/requests/pages_spec.rb]: You could also test which template was rendered.

File: spec/requests/pages_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```
Comment (marclerodrigues) [inline spec/requests/pages_spec.rb]: Use `"`.

File: spec/requests/users_spec.rb (hunk 2 above)
Comment (renatamarques97) [reply spec/requests/users_spec.rb]: ok, got it

File: spec/routing/pages_routing_spec.rb (hunk 1 above)
Comment (renatamarques97) [reply spec/routing/pages_routing_spec.rb]: yes, you're right, i'll change it

File: spec/requests/pages_spec.rb (hunk 4 above)
Comment (renatamarques97) [reply spec/requests/pages_spec.rb]: ok, thanks for the tip

File: spec/requests/pages_spec.rb (hunk 3 above)
Comment (renatamarques97) [reply spec/requests/pages_spec.rb]: alright, i'll do it

File: spec/features/pages/pages_spec.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe 'home page', type: :feature do
+  describe 'texts in home page' do
+
+  	before { visit root_path }
+
+  	context 'match texts in home page' do
+      it 'expect correct title' do
+	       expect(page).to have_content('Projeto Carambolo!')
```
Comment (marclerodrigues) [inline spec/features/pages/pages_spec.rb]: Indentation issues.

File: app/views/pages/home.html.erb [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+      <a class="down" href="#about" data-scroll><i class="icon fa fa-chevron-down" aria-hidden="true"></i></a>
+    </div>
+
+    <section id="about">
+      <div class="user-details">
+        <h1> Sobre </h1>
+        <h2> <%= I18n.t 'site.description' %> </h2>
+      </div>
+       <div class="tech">
+         <p>O projeto surgiu em um encontro do Teresina Hacker Clube, chamado "Nerdices Randômicas,
```
Comment (marclerodrigues) [inline app/views/pages/home.html.erb]: You added i18n for other parts, why not the whole page? What do you think?

File: app/views/pages/home.html.erb [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+          <a class="link" href="#projects" data-scroll>Projetos</a>
+        </div>
+      </div>
+      <a class="down" href="#about" data-scroll><i class="icon fa fa-chevron-down" aria-hidden="true"></i></a>
+    </div>
+
+    <section id="about">
+      <div class="user-details">
+        <h1> Sobre </h1>
+        <h2> <%= I18n.t 'site.description' %> </h2>
```
Comment (marclerodrigues) [inline app/views/pages/home.html.erb]: Add `()` when invoking methods.

File: config/locales/pt-BR.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+pt-BR:
```
Comment (marclerodrigues) [inline config/locales/pt-BR.yml]: You set the languages to pt-BR and EN, could you please add both translations if that not an issue for you? If you can't could you please create a new issue to add the EN part of the translations?

Comment (renatamarques97) [reply config/locales/pt-BR.yml]: sure, I'll do that.

**Comment by sobrinho** [comment]
See the failing specs before merging.

**Pull request merged**